src/GovUk.Education.ExploreEducationStatistics.Admin/Services/PublicationService.cs

```diff
@@ -119,6 +119,11 @@ public class PublicationService : IPublicationService
                         return await ValidateSelectedTopic(updatedPublication.TopicId);
                     }
 
+                    if (publication.Title != updatedPublication.Title)
```

Comment:
We are saying if the topic id is changing skip the check on the title changing (an so also skip the security check on updating the publication title).

I guess this is ok since no user  is likely to be updating the topic id and changing the title at the same time.
Also there's no user who can move a publication between topics who can't update a title as that's for BAU only.

I would have made this independent of the Topic check though, so always check the title changing and do that security check if necessary, even if the topic id has changed and the user can move the topic.

Reply:
I've moved the `Title !=` check into a separate `OnSuccessDo`. I missed that that check wouldn't be executed if the topic had changed.